cmd/operator/operator.go

```diff
@@ -247,11 +257,57 @@ func run(c *cli.Context) {
 	}
 }
 
+func getObjects(objectKinds string) []client.Object {
+	var objs []client.Object
+
+	arr := strings.Split(objectKinds, ",")
+	for _, str := range arr {
+		switch str {
+		case "ConfigMap":
+			objs = append(objs, &v1.ConfigMap{})
+		case "Secret":
+			objs = append(objs, &v1.Secret{})
+		case "Pod":
+			objs = append(objs, &v1.Pod{})
+		case "Event":
+			objs = append(objs, &v1.Event{})
+		case "Service":
+			objs = append(objs, &v1.Service{})
+		case "ServiceAccount":
+			objs = append(objs, &v1.ServiceAccount{})
+		case "PersistentVolume":
+			objs = append(objs, &v1.PersistentVolume{})
+		case "PersistentVolumeClaim":
+			objs = append(objs, &v1.PersistentVolumeClaim{})
+		case "Deployment":
+			objs = append(objs, &appsv1.Deployment{})
+		case "DaemonSet":
+			objs = append(objs, &appsv1.DaemonSet{})
+		case "StatefulSet":
+			objs = append(objs, &appsv1.StatefulSet{})
+		case "ClusterRole":
+			objs = append(objs, &rbacv1.ClusterRole{})
+		case "ClusterRoleBinding":
+			objs = append(objs, &rbacv1.ClusterRoleBinding{})
+		case "Role":
+			objs = append(objs, &rbacv1.Role{})
+		case "RoleBinding":
+			objs = append(objs, &rbacv1.RoleBinding{})
+		case "StorageClass":
+			objs = append(objs, &storagev1.StorageClass{})
+		}
+	}
+
+	log.Infof("%d object kinds will be disabled from cache", len(objs))
+	return objs
+}
+
 func createManager(c *cli.Context, config *rest.Config) (manager.Manager, error) {
 	syncPeriod := defaultResyncPeriod
 	managerOpts := manager.Options{
-		SyncPeriod:         &syncPeriod,
-		MetricsBindAddress: fmt.Sprintf("0.0.0.0:%d", c.Int(flagMetricsPort)),
+		SyncPeriod:            &syncPeriod,
+		MetricsBindAddress:    fmt.Sprintf("0.0.0.0:%d", c.Int(flagMetricsPort)),
+		ClientDisableCacheFor: getObjects(c.String(flagDisableCacheFor)),
```

Comment:
if the input is empty, does it affect anything?

Reply:
it does not, default is empty list.